libdebug/cffi/ptrace_cffi_source.c

```diff
@@ -351,9 +353,10 @@ int cont_all_and_set_bps(struct global_state *state, int pid)
     // continue the execution of all the threads
     t = state->t_HEAD;
     while (t != NULL) {
-        if (ptrace(state->syscall_hooks_enabled ? PTRACE_SYSCALL : PTRACE_CONT, t->tid, NULL, NULL))
-            fprintf(stderr, "ptrace_cont failed for thread %d: %s\\n", t->tid,
+        if (ptrace(state->syscall_hooks_enabled ? PTRACE_SYSCALL : PTRACE_CONT, t->tid, NULL, t->signal))
```

Comment:
Can a signal be received at the same time that  a process hits a breakpoint? In that case I believe that the signal should be delivered during the `SINGLESTEP` at line 335.

Reply:
I had no time to do that, but I plan to address it in the next commits. I just needed something working for now. Thanks for the reminder anyway.